File: Modul 2/python/PRAK204-2310817310010-AkmallullailSyaban.py
angka = input().split()
pi = 22/7
volume = pi * float(angka[0]) * float(angka[0]) * float(angka[1])
luas = 2 * pi * float(angka[0]) * (float(angka[0]) + float(angka[1]))
keliling = 2 * pi * float(angka[0])

print(f"Volume   = {volume:.2f}")
print(f"Luas     = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")

File: Modul 2/C/PRAK204-2310817310010-AkmallullailSyaban.c
#include <stdio.h>
#include <math.h>

#define pi 3.142857

int main(){
    double jari2, tinggi;
    printf("Input\n");
    scanf("%lf %lf", &jari2, &tinggi);  
    double volume = pi * pow(jari2, 2) * tinggi;
    double luas = 2 * pi * jari2 * (jari2 + tinggi);
    double keliling = 2 * pi * jari2;    
    printf("\nOutput\n");   
    printf("Volume   = %.2lf\n", volume);
    printf("Luas     = %.2lf\n", luas);
    printf("Keliling = %.2lf\n", keliling);   
    return 0;
}